examples/flock.jl

```diff
@@ -44,7 +44,7 @@ end
 # a model object using default values.
 function initialize_model(;
     n_birds = 100,
-    speed = 1.5,
+    speed = 1.0,
```

Comment:
```suggestion
    speed = 1.5,
```

is the speed on velocity intended?

Reply:
yes, I think it creates a better visualization but we can keep speed=1.5 if you prefer